Core/tests/WholeProjectRefactorer.cpp

```diff
@@ -4452,13 +4452,13 @@ TEST_CASE("MergeLayers", "[common]") {
     REQUIRE(initialInstances.GetLayerInstancesCount("My other layer") == 1);
   }
 
-  SECTION("Can rename a leaderboard in scene events") {
+  SECTION("Can find and rename leaderboards in a project") {
```

Comment:
The previous description better fit what the test actually covers.

Reply:
Fixed and comment added that we need this for object properties.